compiler-rt/test/asan/TestCases/ill.cpp

```diff
@@ -2,12 +2,18 @@
 //
 // RUN: %clangxx_asan %s -o %t && %env_asan_opts=handle_sigill=0 not --crash %run %t 2>&1 | FileCheck %s --check-prefix=CHECK0
 // RUN: %clangxx_asan %s -o %t && %env_asan_opts=handle_sigill=1 not %run %t 2>&1 | FileCheck %s --check-prefix=CHECK1
-// REQUIRES: x86-target-arch
+// REQUIRES: x86-target-arch && (!MSVC || asan-32-bits)
 
 #ifdef _WIN32
 #include <windows.h>
 #endif
 
+// note: test is limited to i386 only ("asan-32-bits") when using "real" MSVC
+//  see the  REQUIRES: clause above
```

Comment:
This `REQUIRES:` string in the comment here ends up intercepted by `lit`, and can't be parsed, so the test ends up in the category Unresolved, and not executed at all. I guess you at the very least need to drop the `:`, possibly differentiate it further.